pkg/util/sets/uint.go

```diff
@@ -0,0 +1,194 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
```

Comment:
do we really need this, we already have an intset right? why not just use plain ints for declaring capabilities?